games/minimal/mods/testnodes/mod.conf

```diff
@@ -0,0 +1,2 @@
+name = testnodes
```

Comment:
~~test~~ again, and this mod could probably incorporate some of the tests currently in `experimental`?

Reply:
This decision was likely made to avoid mod name conflicts, but also to clarify it's a mod for testing. I don't see an issue there.